cime_config/usermods_dirs/NEON/NIWO/shell_commands

```diff
@@ -2,3 +2,4 @@
 ./xmlchange PTS_LON=254.41676
 ./xmlchange PTS_LAT=40.05236
 ./xmlchange DATM_YR_END=2018
+# remove DATM_YR_END xml change when running with PRISM precipitation
```

Comment:
Additional files that have adjustments:
DATM_YR_END: MLBS, NIWO, MOAB, ONAQ
DATM_YR_START: TEAK, WREF, SJER, YELL (and some non-CONUS sites)

Reply:
My comments on the CDEPS PR are more appropriate here, where we can use logicals similar to this to define start and ends years + the length of transient simulations.

https://github.com/ESCOMP/CTSM/blob/4e126c75a2eaebc6c1c32da4f728bd0891599ff5/cime_config/usermods_dirs/NEON/defaults/shell_commands#L12

Reply:
Thanks @wwieder and @ekluzek . I've added the shell command switch in for the relevant sites.